rpm_spec/core-vm.spec

```diff
@@ -560,15 +552,7 @@ The Qubes core startup configuration for SystemD init.
 /lib/modules-load.d/qubes-core.conf
 /lib/modules-load.d/qubes-misc.conf
 %dir /usr/lib/qubes/init
-/usr/lib/qubes/init/prepare-dvm.sh
-/usr/lib/qubes/init/network-proxy-setup.sh
 /usr/lib/qubes/init/qubes-iptables
-/usr/lib/qubes/init/misc-post.sh
-/usr/lib/qubes/init/misc-post-stop.sh
-/usr/lib/qubes/init/mount-dirs.sh
-/usr/lib/qubes/init/qubes-random-seed.sh
-/usr/lib/qubes/init/qubes-sysinit.sh
-/usr/lib/qubes/init/bind-dirs.sh
```

Comment:
I basically restored the files to their original location to minimize the size of the diff.